package.json

```diff
@@ -30,43 +30,42 @@
   },
   "homepage": "https://github.com/resin-io-modules/resin-multibuild#readme",
   "devDependencies": {
-    "@types/chai-as-promised": "^7.1.0",
-    "@types/js-yaml": "^3.12.1",
-    "@types/mocha": "^5.2.6",
-    "chai": "^4.1.2",
+    "@types/chai-as-promised": "^7.1.2",
+    "@types/js-yaml": "^3.12.2",
+    "@types/mocha": "^5.2.7",
+    "chai": "^4.2.0",
     "chai-as-promised": "^7.1.1",
-    "husky": "^1.3.1",
-    "lint-staged": "^8.1.5",
-    "mocha": "^6.1.3",
+    "husky": "^4.2.3",
+    "lint-staged": "^10.0.7",
+    "mocha": "^6.2.2",
     "ncp": "^2.0.0",
-    "prettier": "^1.16.4",
-    "resin-lint": "^3.0.2",
-    "rimraf": "^3.0.0",
+    "resin-lint": "^3.3.0",
+    "rimraf": "^3.0.2",
     "typedoc": "^0.9.0",
-    "typescript": "^3.4.3"
+    "typescript": "^3.8.2"
   },
   "dependencies": {
-    "@types/bluebird": "3.5.20",
-    "@types/dockerode": "2.5.5",
-    "@types/lodash": "4.14.112",
-    "@types/tar-stream": "1.6.0",
-    "ajv": "^6.10.0",
-    "bluebird": "^3.5.4",
+    "@types/bluebird": "^3.5.29",
+    "@types/dockerode": "^2.5.23",
+    "@types/lodash": "^4.14.149",
+    "@types/tar-stream": "^1.6.1",
+    "ajv": "^6.11.0",
+    "bluebird": "^3.7.2",
     "docker-progress": "^4.0.0",
     "docker-toolbelt": "^3.3.7",
     "dockerfile-template": "^0.1.0",
     "dockerode": "^2.5.8",
-    "fp-ts": "^2.0.0",
-    "io-ts": "^2.0.0",
+    "fp-ts": "^2.5.1",
+    "io-ts": "^2.1.1",
     "io-ts-reporters": "^1.0.0",
     "js-yaml": "^3.13.1",
-    "lodash": "^4.17.4",
+    "lodash": "^4.17.15",
     "resin-bundle-resolve": "^4.2.0",
-    "resin-compose-parse": "^2.0.4",
-    "resin-docker-build": "^1.1.2",
-    "tar-stream": "^2.0.1",
+    "resin-compose-parse": "^2.1.0",
+    "resin-docker-build": "^1.1.3-update-deps-272d8802d61625a4a51ca79feda4a5148d892e34",
```

Comment:
Depends on https://github.com/balena-io/resin-docker-build/pull/38 for the dockerode unpinning